baremetalpxeserver/ansible/baremetalpxeserver.py

```
@@ -89,30 +89,24 @@
-    run_remote_command(command, host_post_info)
-    sys.exit(1)
-
-# name: make sure virtualenv has been setup
-command = "[ -f %s/bin/python ] || virtualenv-2.7 --system-site-packages %s " % (virtenv_path, virtenv_path)
-run_remote_command(command, host_post_info)
 
 # name: install dependencies
 if host_info.distro in RPM_BASED_OS:
     dep_pkg = "dnsmasq nginx nginx-all-modules vsftpd nmap"
+    py3_rpms = ' python3.11 python3.11-devel python3.11-pip libffi-devel openssl-devel'
     if releasever in centos:
```

Ruff (0.11.9)

`centos` may be undefined, or defined from star imports

(F405)